guides/v2.3/install-gde/trouble/tshoot_access-browser.md

```diff
@@ -1 +1,40 @@
-../../../v2.2/install-gde/trouble/tshoot_access-browser.md
```

Comment:
This is a symlink file. Replacing the contents of this file will break the build process.

To properly update this version of the file, please delete the symlink file and create a new file with the updated content.

Reply:
@jcalcaben 
Thank you for explaining.
Done.